README.md

```diff
@@ -1 +1,22 @@
-# Myfirst_Porfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+**Portfolio**
+
+Portfolio website mobile version
+Home section
+![Screenshoot]("./images/screenshot%20of%20mobile%20version.png")
+**Built With**
+- HTML
+- CSS
+- VS Code
+**Authors**
+:silhouette:  **Shahnawaz Ali**
+- GitHub: [@githubhandle](https://github.com/shahnawaza75)
+- Twitter: [@twitterhandle](https://twitter.com/RjShahnawaza75)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shahnawaz-ali-a24b72204)
+ **:handshake: Contributing**
+Contributions, issues, and feature requests are welcome!
+Give a :star:️ if you like this project!
+ **:memo: License**
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] This link is currently broken, please consider fixing it by adding MIT license file into the project